### PR TITLE
Dev > Master for 1.1.0 release

### PR DESCRIPTION
**Pull request by drpatelh**
# nf-core/viralrecon pull request

Many thanks for contributing to nf-core/viralrecon!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/viralrecon branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/viralrecon)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/viralrecon/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: .github/workflows/awstest.yml [hunk 1]
```diff
@@ -6,11 +6,13 @@ on:
   push:
     branches:
       - master
+      - dev
```
Comment (ggabernet) [inline .github/workflows/awstest.yml:9]:
```suggestion
```

File: .github/workflows/awstest.yml [hunk 2]
```diff
@@ -26,5 +28,10 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
           AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
           TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
```
Comment (ggabernet) [inline .github/workflows/awstest.yml:30]:
```suggestion
          AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
          TOWER_ACCESS_TOKEN: ${{secrets.AWS_TOWER_TOKEN}}
          AWS_JOB_DEFINITION: ${{ secrets.AWS_JOB_DEFINITION }}
          AWS_JOB_QUEUE: ${{ secrets.AWS_JOB_QUEUE }}
          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
```

File: .github/workflows/awstest.yml [hunk 3]
```diff
@@ -26,5 +28,10 @@ jobs:
...
-        run: | # Submits job to AWS batch using a 'nextflow-big' instance. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.
-          aws batch submit-job --region eu-west-1 --job-name nf-core-viralrecon --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow-4GiB --container-overrides '{"command": ["nf-core/viralrecon", "-r '"${GITHUB_SHA}"' -profile test_full --outdir s3://nf-core-awsmegatests/viralrecon/results-'"${GITHUB_SHA}"' -w s3://nf-core-awsmegatests/viralrecon/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}, {"name": "NXF_OPTS", "value": "-XX:+UseG1GC"}]}'
+        run: | # Submits job to AWS batch using a 'nextflow-4GiB' instance. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.
+          aws batch submit-job \
+          --region eu-west-1 \
+          --job-name nf-core-viralrecon \
+          --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' \
```
Comment (ggabernet) [inline .github/workflows/awstest.yml:35]:
```suggestion
          --job-queue $AWS_JOB_QUEUE \
```

File: .github/workflows/awstest.yml [hunk 4]
```diff
@@ -26,5 +28,10 @@ jobs:
...
+          --job-definition nextflow-4GiB \
+          --container-overrides '{"command": ["nf-core/viralrecon", "-r '"${GITHUB_SHA}"' -profile test_full --outdir s3://nf-core-awsmegatests/viralrecon/results-'"${GITHUB_SHA}"' -w s3://nf-core-awsmegatests/viralrecon/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}, {"name": "NXF_OPTS", "value": "-XX:+UseG1GC"}]}'
```
Comment (ggabernet) [inline .github/workflows/awstest.yml:37]:
```suggestion
          --container-overrides '{"command": ["nf-core/viralrecon", "-r '"${GITHUB_SHA}"' -profile test_full --outdir s3://'"${AWS_S3_BUCKET}"'/viralrecon/results-'"${GITHUB_SHA}"' -w s3://'"${AWS_S3_BUCKET}"'/viralrecon/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$AWS_TOWER_TOKEN"'"}, {"name": "NXF_OPTS", "value": "-XX:+UseG1GC"}]}'
```

File: .github/workflows/awstest.yml (hunk 1 above)
Comment (MaxUlysse) [inline .github/workflows/awstest.yml:9]: You're doing AWS tests on `dev`?

Comment (drpatelh) [reply .github/workflows/awstest.yml:9]: Ah, I literally just added that in the last commit to `dev` to test if the pipeline is working on AWS. Have removed it already.

**Comment by drpatelh** [comment]
@ggabernet I think I am just going to ignore the AWS tests for this release. The pipeline has already been quite extensively tested on AWS so lets just hope for the best. If things break I will create a patch release.

**Comment by drpatelh** [comment]
@MaxUlysse would you mind giving me a final approval if you are happy please :)

File: .github/workflows/awstest.yml (hunk 1 above)
Comment (drpatelh) [reply .github/workflows/awstest.yml:9]: Will be fixed in https://github.com/nf-core/viralrecon/pull/118

File: .github/workflows/awstest.yml (hunk 2 above)
Comment (drpatelh) [reply .github/workflows/awstest.yml:30]: Will be fixed in https://github.com/nf-core/viralrecon/pull/118

File: .github/workflows/awstest.yml (hunk 3 above)
Comment (drpatelh) [reply .github/workflows/awstest.yml:35]: Will be fixed in https://github.com/nf-core/viralrecon/pull/118

File: .github/workflows/awstest.yml (hunk 4 above)
Comment (drpatelh) [reply .github/workflows/awstest.yml:37]: Will be fixed in https://github.com/nf-core/viralrecon/pull/118

**Comment by drpatelh** [comment]
Amazing! Thank you so much guys!!

**Pull request merged**